python/nano/tutorial/training/pytorch-lightning/lightning_cv_data_pipeline.py

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+# from torchvision import transforms
+from torch.utils.data.dataloader import DataLoader
+import torch
+from torchvision.models import resnet18
+from bigdl.nano.pytorch import Trainer
+from torchmetrics import Accuracy
+import pytorch_lightning as pl
+from bigdl.nano.pytorch.vision.datasets import ImageFolder
```

Comment:
Maybe we should still use `OxfordIIITPet` but import it from `nano`. We can optimize `OxfordIIITPet` as we did for ImageFolder in nano.